web/src/main/resources/ddl/sor/tables.template.cql

```diff
@@ -25,8 +25,19 @@ CREATE TABLE ${table.delta} (
   AND compaction={'sstable_size_in_mb': '160', 'class': 'LeveledCompactionStrategy'}
   AND compression = {'sstable_compression' : 'LZ4Compressor'};
 
+CREATE TABLE IF NOT EXISTS ${table.delta}_v2 (
```

Comment:
As previously noted, probably better to explicitly provide a value rather than assume a "_v2" suffix.